core/shared/src/main/scala/io.buildo.metarpheus/core/extractors/controller.scala

```diff
@@ -12,16 +13,13 @@ package object controller {
       case Mod.Annot(Ctor.Ref.Name("command")) => "post"
     }.get
 
+  private val authType = Type.Name("Auth")
```

Comment:
Can you change this into `private[this]`? (for consistency)

Reply:
🙈